ban/tests/test_models.py

```diff
@@ -349,6 +349,18 @@ def test_housenumber_as_relation():
     }
 
 
+def test_housenumber_municipality_is_cached(mocker, sql_spy):
+    HouseNumberFactory()
+    # Reload the instance to clear property cache from Diff creation.
+    housenumber = models.HouseNumber.first()
+    expected = housenumber.parent.municipality
+    sql_spy.reset_mock()
+    assert housenumber.municipality == expected
+    assert sql_spy.call_count == 1
+    assert housenumber.municipality == expected
+    assert sql_spy.call_count == 1
```

Comment:
@yohanboniface my bad, I read [the wrong part of the diff](https://github.com/BaseAdresseNationale/ban/pull/262/files#diff-9f1e35a38252c8eed20fb8efd5813138R57).